packages/caravan-health/src/waste.test.ts

```diff
@@ -0,0 +1,82 @@
+import { feesToAmountRatio, relativeFeesScore, wasteMetric } from "./waste";
+import { Transaction, FeeRatePercentile } from "@caravan/clients";
+
+describe("Waste metrics that accounts for nature of fees spending for a wallet", () => {
+  it("Relative fees score for transaction with respect to international fiat payment charges", () => {
```

Comment:
This still doesn't quite tell me what you're actually expecting this function to do. Maybe something like:

```suggestion
  it("calculates fee score based on tx fee rate relative to percentile in block a set of send txs were mined", () => {
```

Reply:
You could also make this one test a little more robust by including a non-send tx since I think those are skipped right?

Reply:
yes. correct